### ISSUE TITLE
Unsupported major.minor version 51.0 (Plugin:com.perl5)

### ISSUE THREAD

**Issue by wfox70** [hurricup/Perl5-IDEA#262]
running PhpStorm 9.0 in MAC. 
Getting this error message during loading of the Default Project page.
Deactivation of the plugin helps, but plugin is not working, obviously.
How can I fix this issue?
Thanks!

**Comment by hurricup** [comment]
Check out this issue: https://github.com/jshiell/checkstyle-idea/issues/142

**Comment by wfox70** [comment]
Thanks, hope this works with PHPStorm. This is a third party package, it
might break something else.
Will let you know...

**Comment by hurricup** [comment]
Please note https://github.com/hurricup/Perl5-IDEA/issues/265
At the moment plugin works as intended only in IDEA. Other JB products has some side-effects.

**Comment by wfox70** [comment]
Looks like it worked, but with some modifications.

This is what I did:

Downloaded and installed  Ultimate -
https://download.jetbrains.com/idea/ideaIU-14.1.3-custom-jdk-bundled.dmg

Copied jre folder from /Applications/IntelliJ IDEA 14.app/Contents    to
  /Applications/PhpStorm.app/Contents

Modified Info.plist to:

      <key>JVMVersion</key>

      <string>1.7*,1.8+</string>


All is working great now!

Thanks!

**Comment by hurricup** [comment]
Current version of plugin (0.37 beta) built for 1.6, so should work without additional magic as far as i understand.

**Comment by wfox70** [comment]
I am doing this on MAC and I have PhpStorm 9.0 from JebBrains - that is a
different product from JIDEA 14.
Earlier version of plugin did not work for me before on MAC.
I do have the latest version now 0.37, which I downloaded after switching
to custom JRE that is bundled with IdeaJ14.1.3.
See attached screenshot from the time when I reported the issue.

I followed the recommendation from you about this issue:
jshiell/checkstyle-idea#142
<https://github.com/jshiell/checkstyle-idea/issues/142>
and it works now, as described in my last e-mail.

Sharing with you in case if someone else would have the same issues.

Thanks for the plugin! It makes work with Perl much easier!
